docs/source/getting_started/installation/ai_accelerator/hpu-gaudi.inc.md

```diff
@@ -63,8 +63,7 @@ To build and install vLLM from source, run:
 ```console
 git clone https://github.com/vllm-project/vllm.git
 cd vllm
-pip install -r requirements-hpu.txt
-python setup.py develop
+pip install -v -e .
```

Comment:
do you still need to install "requirements-hpu.txt"?

Reply:
Nope